boards/arm/mimxrt1010_evk/doc/index.rst

```diff
@@ -191,6 +191,107 @@ see the following message in the terminal:
+| CORE_CLK_ROOT              | 500 MHz         |
++----------------------------+-----------------+
+| IPG_CLK_ROOT               | 125 MHz         |
++----------------------------+-----------------+
+| PERCLK_CLK_ROOT            | 62.5 MHz        |
++----------------------------+-----------------+
+| FLEXSPI_CLK_ROOT           | 90 MHz          |
++----------------------------+-----------------+
+| SPDIF0_CLK_ROOT            | 30 MHz          |
++----------------------------+-----------------+
+| FLEXIO1_CLK_ROOT           | 30 MHz          |
++----------------------------+-----------------+
+| SAI1_CLK_ROOT              | 41.53 MHz       |
++----------------------------+-----------------+
+| SAI3_CLK_ROOT              | 41.53 MHz       |
++----------------------------+-----------------+
+| LPI2C_CLK_ROOT             | 10 MHz          |
++----------------------------+-----------------+
+| UART_CLK_ROOT              | 80 MHz          |
++----------------------------+-----------------+
+| ADC_ALT_CLK                | 40 MHz          |
++----------------------------+-----------------+
+| LPSPI_CLK_ROOT             | 90 MHz          |
++----------------------------+-----------------+
+| TRACE_CLK_ROOT             | 99 MHz          |
++----------------------------+-----------------+
+| CKIL_SYNC_CLK_ROOT         | 32.768 kHz      |
++----------------------------+-----------------+
+| Clock 1M output            | 1 MHz           |
++----------------------------+-----------------+
+| Clock 24MHz output         | 24 MHz          |
++----------------------------+-----------------+
+| CLKO1_CLK                  | Inactive        |
++----------------------------+-----------------+
+| CLKO2_CLK                  | Inactive        |
++----------------------------+-----------------+
+| ENET_500M_CLK              | 500 MHz         |
++----------------------------+-----------------+
+| USBPHY PLL clock           | 480 MHz         |
++----------------------------+-----------------+
+| GPT1 high frequency clock  | 62.5 MHz        |
++----------------------------+-----------------+
+| GPT2 high frequency clock  | 62.5 MHz        |
++----------------------------+-----------------+
+| SAI1 MCLK 1                | 41.53 MHz       |
++----------------------------+-----------------+
+| SAI1 MCLK 2                | 41.53 MHz       |
++----------------------------+-----------------+
+| SAI1 MCLK 3                | 30 MHz          |
++----------------------------+-----------------+
+| SAI3 MCLK 1                | 41.53 MHz       |
++----------------------------+-----------------+
+| SAI3 MCLK 2                | Inactive        |
++----------------------------+-----------------+
+| SAI3 MCLK 3                | 30 MHz          |
++----------------------------+-----------------+
+| SPDIF0_EXTCLK              | Inactive        |
++----------------------------+-----------------+
+| MQS MCLK                   | 41.53 MHz       |
++----------------------------+-----------------+
```

Comment:
Just a comment here:
IIUC, lot of peripheral clocks are enabled, while we'd expect them to be disabled at start up if not used (in line with the default board configuration). This being said, there is no recommendation on this today, so that's a maintainer's call here.

Reply:
The clock is being setup here, it's use is disabled and enabled by the driver whenever needed.